app/views/posts/index.html.erb

```diff
@@ -20,12 +20,14 @@
                         <span class="posts-count"><p class="count">Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></p></span>
                     </div>
                 </div>
+                <% if post.comments.any? %>
                 <div class="posts-comment">
                     <% post.comments.each do |comment| %>
                         <p class="comments"><%= comment.author.name %> : <%= comment.text %></p>
                     <% end %>
                 </div>
             <% end %>
         </div>
+        <% end %>
     </div>
```

Comment:
- [ ] [optional] @muneeb3123 it would be nice if you added a link to add a new post page, also a link in your users' page to the user show page, this will make it easier for the user to navigate in your app.